.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  // not all of us have IntelliJ IDEA
+  "editor.formatOnSave": false,
```

Comment:
Yes, but sorry, I doubt that this file will be merged :)

Reply:
you ok if I add it to .gitignore ? I use vscode but don't want to mess up the formatting standards you have (semicolons, etc )

Reply:
I removed the file from the pull request, but added a line in .gitignore